puppet/modules/kubernetes/manifests/kubelet.pp

```diff
@@ -53,9 +53,12 @@
   Array[String] $systemd_requires = [],
   Array[String] $systemd_after = [],
   Array[String] $systemd_before = [],
+  String $config_file = "${kubelet_dir}/kubelet-config.yaml",
```

Comment:
This will point to `/var/lib/kubelet/kubelet-config.yaml`. IMHO this is not the correct location to put the kubelet config. Every config should go somewhere in `/etc/...`

Reply:
Not sure, @simonswine what do you think?

Reply:
It should be /etc/